bpfd/src/serve.rs

```diff
@@ -98,6 +97,12 @@ pub async fn serve(
         }
     };
     if csi_support {
+        const VERSION: &str = env!("CARGO_PKG_VERSION");
+        let node_id = std::env::var("KUBE_NODE_NAME")
+            .expect("cannot start bpfd csi driver KUBE_NODE_NAME not set");
+        let storage_manager =
```

Comment:
```suggestion
            .expect("cannot start bpfd csi if driver KUBE_NODE_NAME not set");
```

Just curious as to why that is?